copy_env

```diff
@@ -19,7 +31,20 @@ def config_from_raw(raw_config)
   config
 end
 
-raw_config = `heroku config -r staging`
+def fetch_config(source)
+  if source == 'heroku'
+    `heroku config -r staging`
+
+  elsif source == 'hokusai'
+    `hokusai staging env get`.gsub(/=/, ': ')
```

Comment:
Hokusai returns the env in this format:

```
my_key=my_value
```

as opposed to Heroku's

```
my_key: my_value
```

Here I rewrite Hokusai's output to match Heroku's, so that the existing `config_from_raw` parsing logic will just work.